src/local-editor-service.ts

```diff
@@ -98,21 +97,45 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModules = document.getByKind('dom-module');
+      if (domModules.size > 1) return;
+      const domModule = domModules.values().next().value;
+      if (!domModule || !domModule.id) return;
+      const outerElement = document.getOnlyAtId('element', domModule.id);
+      if (!outerElement) return;
+      const sortPrefixes = this._createSortPrefixes(outerElement);
+      const innerElement = document.getOnlyAtId('element', location.element.nodeName)!;
+      const innerAttribute = innerElement.attributes.find((value) => value.name === location.attribute)!;
```

Comment:
Please run `npm run format`

I think you probably can configure atom to automatically format with clang on save too.